avaframe/com1DFAPy/runCom1DFA.py

```diff
@@ -95,7 +95,7 @@ def runCom1DFAPy(avaDir='', cfgFile='', relTh='', flagAnalysis=True):
     workDir, outDir = inDirs.initialiseRunDirs(avalancheDir, modName)
     # create particles, create resistance and
     # entrainment matrix, initialize fields, get normals and neighbours
-    particles, fields, Cres, Ment = com1DFA.initializeSimulation(cfgGen, relRaster, dem)
+    particles, fields, Cres, Ment = com1DFA.initializeSimulation(cfgGen, relRaster, dem, avalancheDir)
```

Comment:
Why ist the extra avalancheDir needed? should be in cfgGen, right?

Reply:
ava dir is in cfg not in cfg Gen but I ll try to change it